etc/profile-a-l/firefox.profile

```diff
@@ -14,6 +14,14 @@ mkdir ${HOME}/.mozilla
 whitelist ${HOME}/.cache/mozilla/firefox
 whitelist ${HOME}/.mozilla
 
+# Add the next 5 lines to your firefox.local if you want Firefox to gain
+# access to Gnome extensions (extensions.gnome.org) via browser connector
+#dbus-user.talk org.gnome.ChromeGnomeShell
+#dbus-user.talk org.gnome.Shell
+#dbus-user.talk ca.desrt.dconf
+#dbus-user.talk org.gtk.vfs.Daemon
+#include allow-python3.inc
```

Comment:
Same, and I would just say "the next/following lines". Including a number means that if someone add a new line (e.g. `dbus-user.talk org.gnome.ChromeGnomeShell2`) this must be updated too to be correct.